File: CS/T449201/Controllers/HomeController.cs
using DevExpress.Web.Mvc;
using DevExpress.Web.Office;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace T449201.Controllers {
    public class HomeController : Controller {
        // GET: Home
        public ActionResult Index() {
            return View();
        }

        public ActionResult SpreadsheetPartial() {
            return PartialView("_SpreadsheetPartial");
        }
        public ActionResult SpreadsheetCustomPartial() {
            string documentID = SpreadsheetExtension.GetDocumentId("Spreadsheet");
            SpreadsheetExtension.CloseDocument("Spreadsheet");
            return SpreadsheetExtension.Open("Spreadsheet", documentID);

        }
    }
}

File: VB/T449201/Controllers/HomeController.vb
Imports DevExpress.Web.Mvc
Imports DevExpress.Web.Office
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace T449201.Controllers
    Public Class HomeController
        Inherits Controller

        ' GET: Home
        Public Function Index() As ActionResult
            Return View()
        End Function

        Public Function SpreadsheetPartial() As ActionResult
            Return PartialView("_SpreadsheetPartial")
        End Function
        Public Function SpreadsheetCustomPartial() As ActionResult
            Dim documentID As String = SpreadsheetExtension.GetDocumentId("Spreadsheet")
            SpreadsheetExtension.CloseDocument("Spreadsheet")
            Return SpreadsheetExtension.Open("Spreadsheet", documentID)

        End Function
    End Class
End Namespace